packages/components/Checkbox/src/Checkbox.tsx

```diff
@@ -19,6 +21,14 @@ export const Checkbox = compose<CheckboxType>({
     required: Text,
   },
   useRender: (userProps: CheckboxProps, useSlots: UseSlots<CheckboxType>) => {
+    // Removal of tokens which are not supported for Android
+    {
+      Platform.OS === 'android' &&
```

Comment:
Adding and label and removal of rest of props. which are makes deprecated for android seem okay for unblocking the PR while the right engineering handling for unsupported properties come to an agreement state ( which I think could incline towards not removed). Could you log a tracking item to re-look into this ( both for checkbox from design, as well as engineering code for unimplemented platforms?).

Reply:
Yes , sounds good.

Reply:
I feel there is no advantage of removing the props this way,
We aren't stopping the user in any way from passing the props and even if these props were not removed, since the token files do not contain any specific tokens for the states associated with these props they will still not be applied anyway so it is already equivalent to props not being passed in the first place. ( Eg. Say size is not supported, then 'medium', 'large' would not have any token values specified for android, this would mean it would pick the default values)

Reply:
There are multiple ways of achieving the same implementation in this case, and there seems no clear winner. I would personally incline towards an explicit code which can be read to understand what's being removed, then an inferred code - which could or could not behave as expected in future. It's easier to change a token file in future by someone else, w/o understanding that a specific use case relied on it.